sdk/src/main/java/com/hcaptcha/sdk/HCaptchaWebView.java

```diff
@@ -30,8 +30,8 @@ public HCaptchaWebView(@NonNull Context context, @Nullable AttributeSet attrs, i
     /**
      * Workaround for crash in WebViewChromium
      * Details:
-     *  - https://stackoverflow.com/questions/58519749
-     *  - https://github.com/zulip/zulip-mobile/issues/4051#issuecomment-616855833
+     *  - <a href="https://stackoverflow.com/questions/58519749">...</a>
+     *  - <a href="https://github.com/zulip/zulip-mobile/issues/4051#issuecomment-616855833">...</a>
```

Comment:
Should we add text instead of `...`?

Reply:
Good point. But I cannot propose something better than: Issue investigation p1, Issue investigation p2

Reply:
Issue 1 and Issue 2 sounds good to me.

Reply:
They are both related to the same issue:
 - SO link describes the issue and suggests a workaround
 - Second one is more about conditions/versions that are affected